stac_fastapi/pgstac/stac_fastapi/pgstac/core.py

```diff
@@ -40,7 +40,8 @@ async def all_collections(self, **kwargs) -> List[Collection]:
                     collection_id=coll["id"], request=request
                 ).get_links()
                 linked_collections.append(coll)
-        return linked_collections
+        collection_list = Collections(collections=collections or [], links={})
```

Comment:
Links should be a list

Reply:
Good catch. Which reminds me that we probably need something here rather than simply having the placeholders I added...

Reply:
I *believe* I've corrected this mistake satisfactorily. Here's `/collections` from the docker-compose server:

```javascript
{
   "collections":[
      {
         "type":"Collection",
         "id":"joplin",
         "stac_extensions":[
            
         ],
         "stac_version":"1.0.0",
         "title":null,
         "description":"This imagery was acquired by the NOAA Remote Sensing Division to support NOAA national security and emergency response requirements. In addition, it will be used for ongoing research efforts for testing and developing standards for airborne digital imagery. Individual images have been combined into a larger mosaic and tiled for distribution. The approximate ground sample distance (GSD) for each pixel is 35 cm (1.14 feet).",
         "keywords":null,
         "license":"public-domain",
         "providers":null,
         "summaries":null,
         "extent":{
            "spatial":{
               "bbox":[
                  [
                     -94.6911621,
                     37.0332547,
                     -94.402771,
                     37.1077651
                  ]
               ]
            },
            "temporal":{
               "interval":[
                  [
                     "2000-02-01T00:00:00Z",
                     "2000-02-12T00:00:00Z"
                  ]
               ]
            }
         },
         "links":[
            {
               "rel":"self",
               "type":"application/json",
               "href":"http://localhost:8081/collections/joplin"
            },
            {
               "rel":"parent",
               "type":"application/json",
               "href":"http://localhost:8081/"
            },
            {
               "rel":"items",
               "type":"application/geo+json",
               "href":"http://localhost:8081/collections/joplin/items"
            },
            {
               "rel":"root",
               "type":"application/json",
               "href":"http://localhost:8081/"
            }
         ]
      }
   ],
   "links":{
      
   }
}
```

Reply:
Deleted a prior comment as I spotted a flaw when I posted the example JSON... Will be updating shortly